public/app/slider-theme.css

```
@@ -89,404 +118,486 @@
+    var(--vip-accent)       calc(var(--pct, 50%) - 4%),
+    var(--vip-accent-tip)   var(--pct, 50%),
+    var(--vip-track-empty)  calc(var(--pct, 50%) + 0.15%),
+    rgba(255,255,255,0.045) 100%
+  );
+
+  /* Recessed depth + ambient glow that reads as the fill glowing */
+  box-shadow: inset 0 1px 3px rgba(0, 0, 0, 0.65),
+              0 0 10px rgba(255, 42, 61, 0.14);
+
+  transition: height     0.14s ease,
+              box-shadow 0.20s ease;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)